lib/generators/templates/landable.rb

```diff
@@ -51,6 +51,32 @@
 
   # If you want to save a different UserAgent if the request.user_agent is blank, set it here
   # config.blank_user_agent_string = 'blank'
+
+  # Uncomment to enable publishing of events to an amqp messaging service.
+  # This feature requires the mounting application to configure a messenger
+  # class which includes a publish method.
+  # the EventPublisher class will only attempt to send a message if both
+  #   amqp_enable and amqp_messaging_service are set.
+  # Let landable know to use amqp publishing of the event message
+  # config.amqp_enable = true
+  #
+  # Let landable know what messaging class to use.
+  # This class must include a publish method.
+  # the publish method should accept a text message body.
+  # config.amqp_messaging_service = YourMessagingClass.new(params)
+  #
+  # Set the site segment that will be appended to your messages.
+  # config.amqp_site_segment = 'mybrand:myproduct:myapp'
+  #
+  # Set up the path to event_type mapping for the paths that you want to
+  # publish messages for. An example configuration is below:
+  # config.amqp_event_mapping =
+  #   {
+  #    '/my_path' => { 'GET' => 'Customer Landed',
+  #                    'POST' => 'Customer Submitted',
+  #                    'DELETE' => 'Customer Left'
+  #                  }
+  #   }.freeze
```

Comment:
fixed.  made sense -  allows us to more easily manage these settings as a single collection. :-)